modules/images/webp-uploads/settings.php

```diff
@@ -50,7 +50,7 @@ function webp_uploads_add_media_settings_field() {
 function webp_uploads_generate_webp_jpeg_setting_callback() {
 	?>
 	</td>
-	<td class="td-full">
+	<td class="<?php echo ! is_multisite() ? esc_attr( 'td-full' ) : ''; ?>">
```

Comment:
Instead of this hack, we should simply not render the entire `</td><td class="td-full">` markup when using multisite so that it fits into the default layout out of the box. This way all those CSS hacks below aren't needed either, and we just render the field like it is by default.

Reply:
@felixarntz Makes sense. I have added these changes in the latest commit.